integration_tests/openapi/__init__.py

```diff
@@ -0,0 +1,92 @@
+from typing import TypedDict
```

Comment:
@VishnuSanal , we should add integration tests in the PR

Reply:
for each functions you need to make a test function for example:
```
def test_docs_handler(openapi_instance):
    html_response = openapi_instance.docs_handler()
    assert isinstance(html_response, str)
    assert "swagger.html" in html_response

def test_json_handler(openapi_instance):
    json_response = openapi_instance.json_handler()
    assert isinstance(json_response, str)
    assert "openapi" in json_response
    assert "info" in json_response
    assert "paths" in json_response
```

Reply:
another example:
@pytest.fixture
def openapi_instance():
    return OpenAPI()

def test_openapi_initialization(openapi_instance):
    spec = openapi_instance.openapi_spec
    assert spec["openapi"] == "3.0.0"
    assert spec["info"]["title"] == "Robyn API"
    assert spec["info"]["version"] == "1.0.0"
    assert "paths" in spec
    assert "components" in spec
    assert "servers" in spec
    assert "externalDocs" in spec

Reply:
I have added the tests for all functions -- PTAL. :) btw, thanks for the code!!